gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageGrpcReadChannel.java

```diff
@@ -351,17 +395,43 @@ private static GoogleCloudStorageItemInfo getObjectMetadata(
               RetryDeterminer.SOCKET_ERRORS,
               IOException.class,
               Sleeper.DEFAULT);
+      publishMetric(
+          resourceId,
+          metricsPublisher,
+          GET_OBJECT_METADATA,
+          SUCCESS,
+          JSON,
+          null,
+          "",
+          LATENCY_MS,
+          (System.currentTimeMillis() - startTime));
```

Comment:
This seems like too verbose, can we have some helpers that do not need so many parameters?